### ISSUE TITLE
Improve Web UI accessibility

### ISSUE THREAD

**Issue by sanposhiho** [kubernetes-sigs/kube-scheduler-simulator#91]
The current Web UI is mostly created by me and I don't have any knowledge about accessibility. 😓 
So, it has non-good accessibility and should be improved.

This is a very vague issue. If you have any ideas for improvement, please comment.

/kind feature

**Comment by sanposhiho** [comment]
- control the confirmation dialog with the enter or esc button.

**Comment by sanposhiho** [comment]
ganna close this since there is no clear direction.

/close
